sysdig/template.go

```diff
@@ -0,0 +1,97 @@
+package sysdig
```

Comment:
this is a pretty generic filename, could it be called something more descriptive like `cfn_preprocess_template.go`?

a comment block at the top of this file explaining the high level purpose of these functions would be useful as well